cloudify_aws/rds/tests/test_option_group.py

```diff
@@ -239,7 +243,7 @@ def test_delete_unexpected_client_error(self):
             )
 
         self.assertEqual(
-            str(error.exception), (
+            text_type(error.exception), (
```

Comment:
Same question. No change is necessary, just curious.

Reply:
Same as above